From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Satyrn <satyrn@disroot.org>
Date: Mon, 6 Dec 2021 00:22:30 +0000
Subject: [PATCH] Accept EULA

Satyrn, owner of GalaxiaMC and creator of Interstellar, accepts the Minecraft EULA with this patch.

diff --git a/src/main/java/net/minecraft/server/Main.java b/src/main/java/net/minecraft/server/Main.java
index 36a1d58dd238c835ce3cc9c2376a86a04915d098..70957518564b9f9420fea9894747b10f4d5d113b 100644
--- a/src/main/java/net/minecraft/server/Main.java
+++ b/src/main/java/net/minecraft/server/Main.java
@@ -130,6 +130,7 @@ public class Main {
                 System.err.println( "If you do not agree to the above EULA please stop your server and remove this flag immediately." );
             }
             // Spigot End
+            eulaAgreed = true; // Interstellar
             if (!eula.hasAgreedToEULA() && !eulaAgreed) { // Spigot
                 Main.LOGGER.info("You need to agree to the EULA in order to run the server. Go to eula.txt for more info.");
                 return;
